python/pytests/golden/zip_max_test/test_zip_max.jsonl

```diff
@@ -0,0 +1,4 @@
+{"_time":"2021-01-01T00:00:00.000000000","_key":"A","a":5.7,"b":1.2,"a_zip_max_b":5.7}
+{"_time":"2021-01-01T00:00:00.000000000","_key":"A","a":6.3,"b":0.4,"a_zip_max_b":6.3}
+{"_time":"2021-01-02T00:00:00.000000000","_key":"B","a":null,"b":3.7,"a_zip_max_b":null}
+{"_time":"2021-01-03T00:00:00.000000000","_key":"A","a":13.2,"b":null,"a_zip_max_b":null}
```

Comment:
based on our docs (not the tests), I would have expected the result on line 3 to be `3.7` and not null.

https://kaskada.io/docs-site/kaskada/main/fenl/catalog.html#zip_max  Specfically: 
> If a or b are null, then b will be returned.

Are the docs wrong or is there a bug in the code?

Reply:
I'm not sure? My gut says the docs are wrong, since `null` could be interpreted as "unknown" (any value) in SQL, which means that we don't know what the maximum is. On the other hand, I could see value in "take the greatest of these 3, ignoring any nulls". What does SQL do with `least` and `greatest`? Let's name it based on those and file a bug if the behavior doesn't match?

Reply:
per this text, https://modern-sql.com/caniuse/greatest-least the sql standard says that nulls should always pass through.
 
however seems like numerous dialects ignore them. our code currently follows the standard, so let's leave it as is for now.  eventually we might want to add an option to ignore them.